dvision:
  field_features: 
    all_types: ["t", "x", "l", "penalty_mark", "goal_post"]
    # all_types: ["x", "l", "penalty_mark", "goal_post"]
    t: [
        0,  300,
        0, -300,
      450,  250,
      450, -250,
     -450,  250,
     -450, -250,
    # 2020~ ver.
      450,  150,
      450, -150,
     -450,  150,
     -450, -150,
    # ~2020 ver.
    ]
    x: [
        0,   75,
        0,  -75,
    ]
    l: [
      450,  300,
      450, -300,
     -450,  300,
     -450, -300,
    # 2020~ ver.
      250,  250,
      250, -250,
     -250,  250,
     -250, -250,
      350,  150,
      350, -150,
     -350,  150,
     -350, -150,
    # ~2020 ver.
      # 350,  250,
      # 350, -250,
     # -350,  250,
     # -350, -250,
    ]
    penalty_mark: [
      300,    0,
     -300,    0,
      # 240,    0,
     # -240,    0,
    ]
    goal_post: [
      450,  130,
      450, -130,
     -450,  130,
     -450, -130,
    ]
